simpleclient_jetty/src/main/java/io/prometheus/client/jetty/JettyStatisticsCollector.java

```diff
@@ -1,18 +1,17 @@
 package io.prometheus.client.jetty;
```

Comment:
We're not getting rid of the older one anytime soon, and I don't want to maintain two copies of identical classes as they're likely to drift.